vita3k/gui/src/themes.cpp

```diff
@@ -475,30 +475,30 @@ void draw_start_screen(GuiState &gui, EmuEnvState &emuenv) {
     ImGui::PopFont();
 
     ImGui::PushFont(gui.large_font);
-    const auto DEFAULT_LARGE_FONT_SCALE = ImGui::GetFontSize() / (116.f * SCALE.x);
-    const auto LARGE_FONT_SIZE = (116.f * SCALE.x) * DEFAULT_FONT_SCALE;
-    const auto PIX_LARGE_FONT_SCALE = (96.f * SCALE.x) / ImGui::GetFontSize();
+    const auto DEFAULT_LARGE_FONT_SCALE = ImGui::GetFontSize() / (116.f * SCALE.y);
+    const auto LARGE_FONT_SIZE = (116.f * SCALE.y) * DEFAULT_FONT_SCALE;
+    const auto PIX_LARGE_FONT_SCALE = (96.f * SCALE.y) / ImGui::GetFontSize();
```

Comment:
Is there a reason for using SCALE.y instead of SCALE.x ?

Reply:
font size relate principaly high